.idea/compiler.xml

```diff
@@ -29,8 +29,16 @@
         <module name="killbill-plugin-api-retry" />
         <module name="killbill-plugin-api-routing" />
       </profile>
+      <profile default="false" name="Annotation profile for catalog" enabled="true">
+        <sourceOutputDir name="target/generated-sources/annotations" />
+        <sourceTestOutputDir name="target/generated-test-sources/test-annotations" />
+        <outputRelativeToContentRoot value="true" />
+        <processorPath useClasspath="true" />
+        <module name="catalog" />
+      </profile>
     </annotationProcessing>
     <bytecodeTargetLevel>
+      <module name="catalog" target="1.6" />
```

Comment:
Yup, as well as lines 37 (module name) and 32 (profile name). IDEA should have generated this for you. Did you rename the module maybe later on?